backend/src/main/java/team15/airbnb/accommodation/infrastructure/AccommodationRepository.java

```diff
@@ -0,0 +1,24 @@
+package team15.airbnb.accommodation.infrastructure;
+
+import org.springframework.stereotype.Repository;
+import team15.airbnb.accommodation.domain.Accommodation;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+
+@Repository
+public class AccommodationRepository {
```

Comment:
아래처럼 사용하지 않으시고 직접 구현하셨네요. 학습이 목적이신거죠?? 아마 김영한 님 강의의 느낌이..?

```suggestion
public class AccommodationRepository extends JpaRepository<Accommodation, Long> {
```

Reply:
JPA 학습을 위하여 Spring Data JPA 방식(Repository 인터페이스 상속)이 아닌, 직접 구현의 방식을 택하였습니다.